packages/plugin-infra/src/types.ts

```diff
@@ -305,14 +305,36 @@ export namespace Plugin.SNSAdaptor {
     // #endregion
 
     export interface ApplicationEntry {
+        ID: string
```

Comment:
The plugin should not need to provide its plugin ID again in this field. When you use ApplicationEntry, you should also bring the ID instead of repeat it in the plugin config.

Reply:
No, ApplicationEntry ID is not identify to plugin ID, since there're two ApplicationEntries: ITO and ITO Claim defined in ITO plugin.

Reply:
OK, so you should change the field to something else like `ApplicationEntryID` and explain why it is not the same as PluginID.

Reply:
done